frontend/src/components/LoginForm.jsx

```diff
@@ -146,8 +142,7 @@ const LoginForm = () => {
                 container
                 direction="column"
                 justify="center"
-                alignItems="center"
-              >
+                alignItems="center">
```

Comment:
Some of these changes (like this one) are just due to difference in prettier config and are not actual changes